### PR TITLE
Feature/unt t5605 initial development

### PR DESCRIPTION
**Pull request by nishchal-v**

### CODE REVIEW

File: Source/SSTable/SSTableView.swift [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+                } else if xSwipePercentage >= thirdSwipeThreshold {
+                    /// leading options
+                    /// third swipe action ( if exists othewise second swipe)
+                    singleOptionRevealer(leading: true, index: 0)
+                }
+                previousIndexPaths = indexPaths
+                previousChangeInLocation = locationInTableView
+   
+                /// Commented code for scrolling with selecting
+//                if locationInParentView.y >= (self.bounds.maxY * (100 - downScrollThreshold) / 100)  {
```
Comment (mobile-simformsolutions) [inline Source/SSTable/SSTableView.swift]: remove commented code

File: Source/SSTableCell/SSTableCell.swift [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            view = superview
+
+            if let tableView = view as? SSTableView {
+                self.tableView = tableView
+                return
+            }
+        }
+    }
+    
+
```
Comment (mobile-simformsolutions) [inline Source/SSTableCell/SSTableCell.swift]: remove extra new lines

Comment (nishchal-v) [reply Source/SSTableCell/SSTableCell.swift]: Done

File: Source/SSTable/SSTableView.swift (hunk 1 above)
Comment (nishchal-v) [reply Source/SSTable/SSTableView.swift]: Done

**Pull request merged**